coffee_maker/lib/main.dart

```diff
@@ -34,92 +35,97 @@ class _DemoAppState extends State<DemoApp> {
 
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
+    return CupertinoApp(
+      localizationsDelegates: const <LocalizationsDelegate<dynamic>>[
```

Comment:
Please add comments to highlight that this is needed to get `CupertinoApp` working